tools/hangar/frontend/src/components/tabs/SettingsTab.tsx

```
@@ -181,10 +182,66 @@ function Sidebar({
         </Group>
       </div>
       <ThemeToggle value={theme} onChange={onThemeChange} />
+      <BuildStamp />
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)